makefiles/simulators/Makefile.icarus

```diff
@@ -54,6 +54,8 @@ endif
 
 BUILD_VPI=1
 
+COMPILE_ARGS += -g2012 # Default to latest SystemVerilog standard
```

Comment:
I think this should be conditional on having a new enough version of `iverilog` - this breaks ubuntu 16, which comes with 9.7